File: solutions/6-kyu/your-order-please/javascript/solution.js
function order(words){
  let splitWords = words.split(' ');
  let result = new Array(splitWords.length).fill("");
  for (i = 0; i < splitWords.length; i++) {
    let num = splitWords[i].match(/\d+/);
    result[num - 1] = splitWords[i];
  }
  return result.join(" ");
}

File: solutions/6-kyu/your-order-please/python/solution.py
def order(s):
    split_string = s.split()
    result = ['' for i in range(len(split_string))]
    for word in split_string:
        position = next(int(i) for i in word if i.isdigit())
        result[position-1] = word
    return ' '.join(result)